connector/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryWriteClientModule.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigquery.connector.common;
+
+import com.google.api.gax.retrying.RetrySettings;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.spark.bigquery.SparkBigQueryConfig;
+import com.google.cloud.spark.bigquery.v2.BigQueryDataSourceWriter;
+import com.google.inject.*;
+import org.apache.spark.sql.SaveMode;
+import org.apache.spark.sql.types.StructType;
+
+public class BigQueryWriteClientModule implements Module {
```

Comment:
Change interface reference to `com.google.inject.Module` in order to avoid conflict with `java.lang.Module`